sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -720,9 +721,25 @@ func (host *nodeLanguageHost) execNodejs(ctx context.Context, req *pulumirpc.Run
 	var errResult string
 	// #nosec G204
 	cmd := exec.Command(nodeBin, nodeargs...)
-	if err := copyOutput(cmd); err != nil {
-		return &pulumirpc.RunResponse{Error: fmt.Errorf("copy node output: %w", err).Error()}
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return &pulumirpc.RunResponse{Error: fmt.Errorf("failed to get stdout pipe: %w", err).Error()}
+	}
+	// Copy cmd.Stdout to os.Stdout. Nodejs sometimes changes the blocking mode of its stdout/stderr,
+	// so it's unsafe to assign cmd.Stdout directly to os.Stdout. See `copyOutput`.
```

Comment:
Looks like `copyOutput` isn't relevant here any more (I assume it's still used elsewhere though), so maybe we should update this comment?

Reply:
The description of `copyOutput` has some more details on why we're doing this. I'll change the comment to make it clear the reader should go look at `copyOutput`'s description.